docs/conf.py

```diff
@@ -3,60 +3,20 @@
 # For the full list of built-in configuration values, see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
-# -- Project information -----------------------------------------------------
-# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
-
 project = "canonical-sphinx"
 copyright = "2024, Canonical"
```

Comment:
You should use something similar to `copyright = f"{datetime.today().year}, {author}"` here as well, or it will be outdated quickly. ;)